lib/datadog/tracing/contrib/rails/ext.rb

```diff
@@ -12,6 +12,10 @@ module Ext
           ENV_ANALYTICS_ENABLED = 'DD_TRACE_RAILS_ANALYTICS_ENABLED'
           ENV_ANALYTICS_SAMPLE_RATE = 'DD_TRACE_RAILS_ANALYTICS_SAMPLE_RATE'
           ENV_DISABLE = 'DISABLE_DATADOG_RAILS'
+          SPAN_ROUTE = 'rails.route'
```

Comment:
How are those constants useful?

Reply:
Maybe they're not anymore. I do believe that having all intermediate routing spans is super useful.

Ideally:
- These should probably tag `SCRIPT_NAME` and `PATHINFO` as well.
- I would also add a Rack `UrlMap` instrumentation that creates a `rack.map` span with a `rack.map.path` (not `route` as it's not a route, but that's debatable) for having full trace clarity on routing behaviour observability.